pkg/identityserver/rights.go

```diff
@@ -332,10 +337,6 @@ func (is *IdentityServer) assertGatewayRights( // nolint:gocyclo
 			return err
 		}
 		if len(membershipChains) != len(entityIDs) {
-			// Some memberships were not found.
```

Comment:
At this point, if the user is an admin, and they don't have access via the universal rights, we need to check their chain.

Both not having a chain, or having a chain but not enough rights, are situations in which `errInsufficientRights` is valid.